bin/lowly.js

```diff
@@ -26,13 +30,26 @@ const cli = meow(`
 
 const input = cli.input
 
-function getOutputFileName(input) {
+const getOutputFileName = input => {
   const inputPath = path.parse(input)
   return path.resolve(inputPath.dir, inputPath.name + '-lowly' + inputPath.ext)
 }
 
+const checkType = (buff, file) => {
+  return new Promise((resolve, reject) => {
+    magic.detect(buff, (err, res) => {
+      if (res.indexOf('image') >= 0) {
```

Comment:
Here is only verified if the file is an image. But, not all image formats that are support, only `bmp`, `jpg` and `png`.  So you should verify these formats.